js/plugins/ollama/tests/model_test.ts

```diff
@@ -79,6 +84,14 @@ describe('ollama models', () => {
     });
   });
 
+  it('should successfully return basic response', async () => {
+    const result = await ai.generate({
+      model: 'ollama/test-model',
+      prompt: 'Hello',
+    });
+    assert.ok(result.text === 'The weather is sunny');
+  });
+
   it('should successfully return tool call response', async () => {
```

Comment:
let's figure out why this test isn't passing if it wasn't, instead of changing the test